java-checks/src/main/java/org/sonar/java/checks/tests/AssertionsInTestsCheck.java

```diff
@@ -172,23 +173,6 @@ private static boolean expectAssertion(MethodTree methodTree) {
     return false;
   }
 
-  private static boolean isUnitTest(MethodTree methodTree) {
```

Comment:
shouldn't you also remove the `ASSERTION_INVOCATION_MATCHERS` and `REACTIVE_X_TEST_METHODS` constants from this file, since you duplicated it in `UnitTestUtils`?